src/blocks/blocks/progress-bar/edit.js

```diff
@@ -79,6 +82,20 @@ const ProgressBar = ({
 		);
 	}, [ attributes.percentage, attributes.duration ]);
 
+	useEffect( () => {
+		setCSSVars( blockRef.current, {
+			'--titleColor': attributes.titleColor,
+			'--percentageColor': attributes.percentageColor,
+			'--percentageColorOuter': attributes.percentageColor,
+			'--percentageColorTooltip': attributes.percentageColor,
+			'--percentageColorAppend': attributes.percentageColor,
+			'--backgroundColor': attributes.backgroundColor,
+			'--borderRadius': ( undefined !== attributes.borderRadius ? attributes.borderRadius : 5 ) + 'px',
+			'--height': ( undefined !== attributes.height ? attributes.height : 30 ) + 'px',
+			'--barBackground': attributes.barBackgroundColor
+		});
+	}, [ blockRef.current, attributes ]);
+
```

Comment:
@HardeepAsrani, using JS to set the CSS vars is the only reliable way to make the CSS work in Editor. We need to convert all the Emotion CSS to this method in this situation.

Reply:
If you referring to `css` var, I can not use it just to render the CSS and inject inline.

Reply:
Also, we do not only need to `only load for FSE`. This can be default method for add CSS

Reply:
You can keep it like this working for one block. I'll take a look and let you if I've any input.

Reply:
What do you mean by working for one block?